### ISSUE TITLE
[a11y] 9.1.1.1b - Alternative texts for graphics and objects

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5376]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
Note / "Login“-Page (not part of the test): The image for the logo has a alternative text only the word
"ownCloud". Better would be "Logo ownCloud".

## Status: not applicable
